### DISCUSSION TITLE
Implement of WL-134 RFID

### DISCUSSION THREAD

**Discussion by IchMagFanta** [omersiar/esp-rfid#642] [Ideas]
Hello,
i Love this Projekts and have a feature request for WL-134 RFID reader for animal Tags in V2.

I would love to use this for petdoors and etc.. 

Best regards
Frank

**Comment by matjack1** [comment]
Hey @IchMagFanta the use case for petdoors is very cool! 

Do you have more information on how the sensor works? Do you have code examples that you have tried already? If you want I can help you setting up a pull request.

**Reply by IchMagFanta** [reply]
Hey @matjack1,
Connection via ttl via serial. 
Here is a library that converts tags into plain text. So it could be adjusted quickly like the RDM6300?

https://github.com/Makuna/Rfid134

[RFID reader module.pdf](https://github.com/user-attachments/files/15793388/RFID.reader.module.pdf)

**Reply by matjack1** [reply]
hey @IchMagFanta yes, possibly that library works, but needs to be tested. I would say to open an issue here: https://github.com/esprfid/esp-rfid/issues as we are tracking feature requests and bugs over there. If you or anyone else want to give it a go, I'll be happy to review the code and help you to get that merged :)
